modules/ROOT/partials/depl-examples/ubuntu-compose/shared-setup.adoc

```diff
@@ -0,0 +1,442 @@
+Set the domain names as defined in xref:domain-names[Domain Names].
+
+* `SMPT_xxx` +
+Define these settings according your eMail configuration. With the settings defined, Infinite Scale is able to send notifications to users. If the settings are not defined, Infinite Scale will start, but notifications can't be sent.
+
+[NOTE]
+====
+* When not defining your own domain names, internal evaluation only domain names with self signed certificates are used automatically.
+
+* LetsEncrypt notes:
+
+** We recommend *before using live certificates*, to use the https://letsencrypt.org/docs/staging-environment/[staging environment of Letsencrypt, window=_blank] which you can configure via `TRAEFIK_ACME_CASERVER`. If certificates can be created and are issued by `Fake LE intermediate X1`, you can switch back to issuing valid certificates.
+
+** To trigger cerfificate issuing via LetsEncrypt, it checks, in the request for creating valid certificates, if the response eMail address is valid and continues if so. The eMail address used is defined via the variable `TRAEFIK_ACME_MAIL`. Self-signed certificates are being used if the traefik log contains the message `Contact emails @example.org are forbidden`.
+====
+
+== Start the Compose Setup
+
+When you have finished the configuration, you can start the compose setup by issuing the following command:
+
+[source,bash]
+----
+docker compose up -d --remove-orphans
+----
+
+This command will download all necessary containers and starts up the instance according your settings in the background (flag `-d`).
+
+Check the logs via the `docker logs command`, especially the traefik logs. See the xref:monitor-the-instance[Monitor the Instance] for more details on logging.
+
+If no issues are logged, traefik and LetsEncrypt were able to handle connectivity and domains. In case you have used staging certificates as suggested, down the compose environment, change the setting and restart it. Recheck the traefik logs and when all is fine, you can access your instance, for details see below. 
+
+=== Solving First Startup Issues
+
+If any issues are logged by traefik on first startup with respect to LetsEncrypt like:
+
+* `...unable to generate a certificate for the domains...`, `acme: error: 400` and `acme-challenge`: +
+Check if the ports 80/443 are open in the firewall configured. You can run a test _during running compose_ to test if traefik can be reached on those ports. To do so visit https://letsdebug.net[Let's Debug].
+
+* `...DNS problem: NXDOMAIN looking up A for...` +
+This points to a DNS resolution problem. Check if the domains entered in the DNS and in the `.env` file match. Note that when using wildcard domains on the DNS, the fixed part must match on both sides.
+
+Whenever you had a DNS issue at this stage, you will face follow up issues on a consecutive compose start because the certificae volume now holds invalid data. Therefore the cert volume needs to be deleted:
```

Comment:
```suggestion
Whenever you had a DNS issue at this stage, you will face follow up issues on a consecutive compose start because the certificate volume now holds invalid data. Therefore the cert volume needs to be deleted:
```